pkgs/crypto/deposit_data.go

```diff
@@ -39,7 +39,7 @@ func BuildDepositDataCLI(network core.Network, depositData *phase0.DepositData,
 	depositDataJson := &wire.DepositDataCLI{
 		PubKey:                hex.EncodeToString(depositData.PublicKey[:]),
 		WithdrawalCredentials: hex.EncodeToString(depositData.WithdrawalCredentials),
-		Amount:                spec_crypto.MaxEffectiveBalanceInGwei,
+		Amount:                uint64(spec_crypto.MaxEffectiveBalanceInGwei),
```

Comment:
silly go question...
is this casting needed?

I see `Gwei` is an alias for `uint64` and before everything compiled

Reply:
compiler is complaining `cannot use spec_crypto.MaxEffectiveBalanceInGwei (constant 32000000000 of type phase0.Gwei) as uint64 value in struct literalcompiler`